agent/grpc-external/services/resource/server.go

```diff
@@ -19,6 +19,11 @@ type Config struct {
 	backend  storage.Backend
 }
 
+//go:generate mockery --name Backend --inpackage
+type Backend interface {
+	storage.Backend
+}
+
```

Comment:
This is really clever btw! I've always struggled to figure out how to generate mocks for interfaces in other packages 😍